### ISSUE TITLE
Canvas cursor doesn't change on selected element

### ISSUE THREAD

**Issue by Janpot** [mui/mui-toolpad#1613]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Steps to reproduce 🕹

Steps:

1. grab a component and hover over itself. The cursor doesn't behave the way it does over a non-selected element

https://user-images.githubusercontent.com/2109932/215443579-8bf9d254-5267-4377-b67d-8129f8686dda.mov

### Context 🔦

Marking this as low priority as nothing is blocked by this behavior. It just feels counter intuitive because the cursor doesn't behave predictably when initially starting a drag, and that may set the expectations for the rest of the UX.

**Comment by apedroferreira** [comment]
Shouldn't be too hard to fix, I can take it.

**Comment by apedroferreira** [comment]
Wait how did you manage to have a component stay selected while you drag & drop? 
I just tried this and it always deselects while you drag.

We can fix this anyway just in case though.

**Comment by Janpot** [comment]
>Wait how did you manage to have a component stay selected while you drag & drop?

it's the one I'm dragging

**Comment by prakhargupta1** [comment]
It seems to be fixed now.
